src/core/prompts/tools/read-file.ts

```diff
@@ -1,74 +1,73 @@
 import { ToolArgs } from "./types"
 
 export function getReadFileDescription(args: ToolArgs): string {
-	// Base description without partial read instructions
-	let description = `## read_file
-Description: Request to read the contents of a file at the specified path. Use this when you need to examine the contents of an existing file you do not know the contents of, for example to analyze code, review text files, or extract information from configuration files. The output includes line numbers prefixed to each line (e.g. "1 | const x = 1"), making it easier to reference specific lines when creating diffs or discussing code.`
+	const maxConcurrentReads = args.settings?.maxConcurrentFileReads ?? 15;
+	const isMultipleReadsEnabled = maxConcurrentReads > 1;
+	
+	return `## read_file
+Description: Request to read the contents of ${isMultipleReadsEnabled ? "one or more files" : "a file"}. The tool outputs line-numbered content (e.g. "1 | const x = 1") for easy reference when creating diffs or discussing code. Use line ranges to efficiently read specific portions of large files. Supports text extraction from PDF and DOCX files, but may not handle other binary files properly.
 
-	// Add partial read instructions only when partial reads are active
-	if (args.partialReadsEnabled) {
-		description += ` By specifying start_line and end_line parameters, you can efficiently read specific portions of large files without loading the entire file into memory.`
-	}
+${isMultipleReadsEnabled ? `**IMPORTANT: You can read a maximum of ${maxConcurrentReads} files in a single request.** If you need to read more files, use multiple sequential read_file requests.` : "**IMPORTANT: Multiple file reads are currently disabled. You can only read one file at a time.**"}
 
-	description += ` Automatically extracts raw text from PDF and DOCX files. May not be suitable for other types of binary files, as it returns the raw content as a string.
+${args.partialReadsEnabled ? `By specifying line_range xml tag parameter, you can efficiently read specific portions of large files without loading the entire file into memory.` : ""}
```

Comment:
Can we make this read a little better? Maybe
```suggestion
${args.partialReadsEnabled ? `By specifying line ranges, you can efficiently read specific portions of large files without loading the entire file into memory.` : ""}
```